<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelNAOIP">
         <property name="text">
          <string>NAO IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineIP"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string>File Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileNameLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="StartRecordingButton">
         <property name="text">
          <string>Start Recording</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="image_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>741</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="commandsLabel">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>140</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Commands</string>
    </property>
   </widget>
   <widget class="QFrame" name="commandArea">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>170</y>
      <width>341</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="changeCameraButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Picture</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="hsvCheckBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>98</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>321</width>
       <height>208</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSlider" name="h1Slider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="h2Slider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="s1Slider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="s2Slider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="v1Slider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="v2Slider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="h1Label">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="h2Label">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="s1Label">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="s2Label">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="v1Label">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="v2Label">
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>changeCameraButton</zorder>
    <zorder>pushButton</zorder>
    <zorder>hsvCheckBox</zorder>
    <zorder>h1Slider</zorder>
    <zorder>h2Slider</zorder>
    <zorder>s2Slider</zorder>
    <zorder>s1Slider</zorder>
    <zorder>v1Slider</zorder>
    <zorder>v2Slider</zorder>
    <zorder>h1Slider</zorder>
    <zorder></zorder>
   </widget>
   <widget class="QLabel" name="unbeatablesLogo">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="simulatorName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>771</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Frednator</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
